### DISCUSSION TITLE
Does this support Claude from GCP?

### DISCUSSION THREAD

**Discussion by ankit-tyagi** [cline/cline#260] [Feature Requests]
Does this support Claude from GCP?

**Comment by Pant** [comment]
Yes. You have to specify project name and region, as well as Login from the cli to gcp (thats how i managed to use it)

**Reply by 123hi123** [reply]
hi guys i wanna know how to use i already try for a long time but cant work 
![image](https://github.com/user-attachments/assets/2a884ba0-35d2-42a9-a5f3-856ce9fb2d9b)

**Reply by Ryuu3rs** [reply]
I have the same Issue As you do @123hi123 
0 listed  every time i dont seem to understand what im doing wrong I reread all the documentation and Support docs on Google about setup Asked AI to help me And then re did it all over a few times for the last few days to get this to work

**Reply by 123hi123** [reply]
> I have the same Issue As you do @123hi123 
> 0 listed  every time i dont seem to understand what im doing wrong I reread all the documentation and Support docs on Google about setup Asked AI to help me And then re did it all over a few times for the last few days to get this to work 

I just ignore this recently, I can use gcp via openai capable 
But, the experience is good cause the gcp have low token volume and rpm
